packages/pluggableWidgets/switch-native/src/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="Switch" version="1.1.0" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="Switch" version="1.0.0" xmlns="http://www.mendix.com/clientModule/1.0/">
```

Comment:
This probably went wrong during rebase. It shouldn't go down

Reply:
We will set the version to 1.0.0 and create new separate PR for the changelog